### ISSUE TITLE
Feature Request: Additional Pod Field Support

### ISSUE THREAD

**Issue by Kpudlo** [elementor/elementor#9114]
## Prerequisites

- [x ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
I love that I can select Pods fields as the source for dynamic population! However, I'd love to have the option for additional field support, specifically currency and yes/no fields. 

**Describe the solution you'd like**
Being able to select Pods fields that are set to currency, yes/no, or other formats for dynamic content population.

**Describe alternatives you've considered**
Using Pods Shortcodes

**Additional context**
![image](https://user-images.githubusercontent.com/4432822/65054322-60181680-d93b-11e9-95e9-00d78d848146.png)
![image](https://user-images.githubusercontent.com/4432822/65054380-732ae680-d93b-11e9-8d0a-77a28194c08b.png)

**Comment by kochhase** [comment]
yes that would be fine. so i don´t have to use toolset any more.

**Comment by crazypsycho** [comment]
For yes/no fields, the solution would be very easy.
Just add "boolean" to the array in elementor-pro/modules/dynamic-tags/pods/tags/pods-text.php on line 52.
I think for other field-types it would work the same way.

Yes/no fields would be very useful, because it could used for dynamic conditions or possibly as a css-class with a prefix to highlight some things and surly a lot more.

**Comment by Erudition** [comment]
I'm also looking for Yes/No support.
